supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -212,26 +209,52 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     if (referenceName.length <= 127) referenceName else referenceName.md5Sum
   }
 
+  private[jes] def findGlobOutputs(jobDescriptor: BackendJobDescriptor): Seq[WdlGlobFile] = {
```

Comment:
Does this need to use a `Seq` or `List` in all the types?  It doesn't appear that order is important.